main_test.go

```diff
@@ -341,8 +341,7 @@ func emuCheck(t *testing.T, options compileopts.Options) {
 	}
 	if spec.Emulator != "" {
 		emulatorCommand := strings.SplitN(spec.Emulator, " ", 2)[0]
-		_, err := exec.LookPath(emulatorCommand)
-		if err != nil {
+		if _, err := exec.LookPath(emulatorCommand); err != nil {
```

Comment:
Please don't make unrelated changes in the PR. While this is probably a good idea, it would be better to make a separate cleanup PR instead of putting everything in one huge PR.

Reply:
A nice phrase I picked up somewhere was "clean or fix, not both".